README.md

```diff
@@ -89,6 +89,14 @@ The program can be configured using either a JSON configuration file or environm
   },
   "filesystem": {
     "path": "your_filesystem_path"
+  },
+  "postgresql": {
+    "user": "postgres",
+    "password": "postgres",
+    "host": "localhost",
+    "port": 5432,
+    "database": "coordinator",
```

Comment:
suggestion: perhaps instead of `coordinator` use `delegation_program` since databases have been renamed recently?